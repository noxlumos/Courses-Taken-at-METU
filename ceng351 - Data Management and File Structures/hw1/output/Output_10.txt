
--------------------------------------------------------------
*** Q5 ***
author_id	author_name

--------------------------------------------------------------
