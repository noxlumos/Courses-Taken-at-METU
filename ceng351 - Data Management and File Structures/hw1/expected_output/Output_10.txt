
--------------------------------------------------------------
*** Q5 ***
author_id	author_name
11151	Will Kymlicka
14187	Rengin Isik
29350	Esra Tuncer

--------------------------------------------------------------
